app/models/task.rb

```diff
@@ -9,15 +9,20 @@ class Task < ApplicationRecord
 
   acts_as_tenant(:tenant)
 
+  @@timeout_interval = 0
+
+  def timeout_interval
+    @@timeout_interval
+  end
+
+  def timed_out?
```

Comment:
@hsong-rh If the timeout_interval is 0 would timed_out? return false or true? Or should we make the default timeout_interval to be 2 hours (max value)

Reply:
yes, you are right, I will change it